helmfile/values/falco.yaml.gotmpl

```diff
@@ -109,6 +109,43 @@ customRules:
             container.image.repository=docker.io/elastisys/curl-jq and k8s.pod.name startswith opensearch-
           )
         )
+    - macro: user_known_non_sudo_setuid_conditions
```

Comment:
Consider allowing these types of configs be inserted from sc/wc-config.
But it can also be added like this.

Reply:
It's already possible to add them that way https://github.com/elastisys/compliantkubernetes-apps/blob/main/helmfile/values/falco.yaml.gotmpl#L21-L23